src/js/containers/awardV2/idv/AwardMetaDataContainer.jsx

```diff
@@ -22,28 +22,36 @@ export class AwardMetaDataContainer extends React.Component {
             federalAccountCount: 0
         };
     }
+
     async componentDidMount() {
+        await this.getAwardMetaData();
+    }
+
+    async componentDidUpdate(prevProps) {
```

Comment:
Interesting use of `async`/`await`! Something to consider for other containers as well.